contracts/interfaces/external/ICurvePool/ICurvePool.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+/// @title Curve pool interface
+interface ICurvePool {
+    function token() external view returns (address);
+
+    function coins(uint256 index) external view returns (address);
+
+    function remove_liquidity_one_coin(
```

Comment:
`remove_liquidity_one_coin` function are never used with the Interface (only using the signature). We can remove them if we use directly the signature (literal).